web/main/colors.css

```diff
@@ -6,6 +6,11 @@
   --fire1-color: oklch(70% 0.15 65);
   --fire2-color: oklch(58% 0.2 29);
   --error-color: var(--red-600);
+  --code-inline-bg-color: color-mix(in oklch, var(--neutral-700), var(--neutral-600));
+  --code-inline-color: oklch(100% 0 0);
+  --link-color-hover: oklch(from var(--link-color) l c h / 85%);
```

Comment:
Can we use `opactiy`?

Reply:
Yes, it makes sense. It's definitely a better decision in this case. done